app.js

```diff
@@ -1,26 +1,32 @@
 require('dotenv').config();
 var express = require('express');
 var path = require('path');
-var cookieParser = require('cookie-parser');
 var bodyParser = require('body-parser');
-var logger = require('morgan');
 var mongoose = require('mongoose');
-var shortid = require('shortid');
```

Comment:
Why was shortid removed?

Reply:
It was moved to the bottom of the imports list.